cli/src/main/java/org/infinispan/cli/commands/Quit.java

```diff
@@ -12,7 +12,7 @@
  * @since 10.0
  **/
 @MetaInfServices(Command.class)
-@CommandDefinition(name = "quit", description = "Quits the CLI")
+@CommandDefinition(name = "quit", description = "Quits the CLI", aliases = {"exit", "bye"})
```

Comment:
The aliases are missing from the help menus and the documentation. They also aren't presented as options when pressing tab for suggestions.

Reply:
I added these aliases to the manpage for the quit command. I'm not sure they should be presented as options with tab suggestions but that's not my call.

Reply:
Aliases are not included in tab completion because they would pollute it a bit